pkg/solana/fees/block_history.go

```diff
@@ -310,3 +339,8 @@ func (bhe *blockHistoryEstimator) populateCache(ctx context.Context, loadBatch,
 	bhe.cache.storedBlockRange = bhe.cache.storedBlockRange[excessBlocks:]
 	return nil
 }
+
+func (bhe *blockHistoryEstimator) recordComputeUnitPrice(ctx context.Context, avgOfMedians ComputeUnitPrice) {
+	promBHEComputeUnitPrice.WithLabelValues(bhe.chainID).Set(float64(avgOfMedians))
+	bhe.computeUnitPrice.Record(ctx, int64(avgOfMedians), metric.WithAttributes(attribute.String("chainID", bhe.chainID))) //nolint:gosec // compute unit prices cannot exceed int64 max
```

Comment:
Nitpick: the CU price is actually a `u64` (https://github.com/anza-xyz/agave/blob/67412607f511ded3770031280b6aaf10607713fc/sdk/src/compute_budget.rs#L48-L50), so it is theoretically possible for it to exceed the `int64` range. Is there any chance you can change the gauge type to reflect this? 

I know that that exceeding the `int64` max would be a very high price, so it's very unlikely to ever happen, but I'd still advocate for either having the metric reflect the actual type/range of the value, or at least have the conversion code be fail-proof (even if you just want to floor any higher price back to `int64::max`, that's fine by me)